### PR TITLE
rename to gregorian

### PR DESCRIPTION
**Pull request by pipobscure**
Did the rename and a bit of cleanup in the README.

### CODE REVIEW

**Comment by maggiepint** [comment]
@mj1856

File: README.md [hunk 1]
```diff
@@ -106,58 +106,58 @@ new CivilDate(year, month, day)
...
 ---------------------------------------------------------------------------------------------------
 
-# Object: `CivilTime`
+# Object: `GregorianTime`
 Represents a position on a 24-hour clock.
 
 ### Constructor
 ```js
-new CivilTime(hour, minute[[[, second], millisecond], nanosecond])
+new GregorianTime(hour, minute[[, second], nanosecond])
```
Comment (ljqx) [inline README.md]: seems should be `new GregorianTime(hour, minute[, second[, nanosecond]])`

**Comment by pipobscure** [comment]
Thanks. Hold the PR since I think we may go for ISODate instead.

Ceterum censeo religiones debent delerentur essent propter perniciosas sunt.

**Comment by pipobscure** [comment]
There is no consensus / agreement / ... here

**Pull request closed without merging**